scout/server/utils.py

```diff
@@ -30,6 +31,10 @@ def html_to_pdf_file(html_string, orientation, dpi=96):
         "orientation": orientation,
         "encoding": "UTF-8",
         "dpi": dpi,
+        "margin-top": margins[0],
```

Comment:
👍

Reply:
I've changed the top margin from 2 cm to 1.5. I've also updated the preview in my comment above. It looks better now I think!